pypeclub/OpenPype
# Maya Deadline: Fix Tile Rendering by forcing integer pixel values

## Brief description

This should resolve the issue faced in #3451

## Description

Forces integer pixel values into the Tile Assembly config files.

## Testing notes:

1. Submit Tile Render
2. Ensure tile render assembles correctly
